docs/integration_resources/resources/usd_tutorials/usd_tutorial_05.md

```diff
@@ -0,0 +1,97 @@
+</figure>
+
+3. For traversing the stage, add a new **<tt>UsdStageTraverse</tt>** node to the board and rename it as `TraverseStage`. Connect the **stage** output plug from the `OpenStage` node to the **stage** input plug of the new node. Leave the **primType** input plug set to `Any` and the **traverseAll** input plug set to `False`. The board should look like the following:
+
+<figure>
+    <img src="images/usd_tutorial_05/step_03_t05.png" alt="Added 'TraverseStage' node to the board connected to 'OpenStage'.">
+    <figcaption><b>Figure 3</b>: Added 'TraverseStage' node to the board connected to 'OpenStage'.</figcaption>
+</figure>
+
+4. Execute the current workflow for checking the **prims** output plug from the `TraverseStage` node. The value of the plug should be: `[Usd.Prim(</refSphere>), Usd.Prim(</refSphere/world>), Usd.Prim(</refSphere2>), Usd.Prim(</refSphere2/world>)]`, containing all the prims that build the stage. The board should look like the following:
+
+<figure>
+    <img src="images/usd_tutorial_05/step_04_t05.png" alt="'TraverseStage' node prims plug value.">
+    <figcaption><b>Figure 4</b>: 'TraverseStage' node prims plug value.</figcaption>
+</figure>
+
+5. For filtering the values of the **prims** output plug in the `TraverseStage` node, set the **primType** input plug to `Sphere`. Execute again the node for refreshing the output plug result. The value of the plug once the node is executed again should be: `[Usd.Prim(</refSphere/world>), Usd.Prim(</refSphere2/world>)]`. The board should look like the following:
+
+<figure>
+    <img src="images/usd_tutorial_05/step_05_t05.png" alt="'TraverseStage' node prims plug value filtering by 'Sphere' primtype.">
+    <figcaption><b>Figure 5</b>: 'TraverseStage' node prims plug value filtering by 'Sphere' primtype.</figcaption>
+</figure>
+
+6. In order to activate/deactivate a prim in an specific layer, for the current example, the session layer, add a new **<tt>UsdStageSessionLayerGet</tt>** node and rename it as `SessionLayerGet`. Connect the **stage** output plug from the `OpenStage` node to the **stage** input plug of the new node. The board should look like the following:
+
+<figure>
+    <img src="images/usd_tutorial_05/step_06_t05.png" alt="Added the 'SessionLayerGet' node to the board connected to the 'OpenStage' node.">
+    <figcaption><b>Figure 6</b>: Added the 'SessionLayerGet' node to the board connected to the 'OpenStage' node.</figcaption>
+</figure>
+
+7. Now, setting the edit target of the stage is important, in this way, any modification done will affect specifically to that layer instead of the root layer of the stage. Add a new **<tt>UsdStageEditTargetSet</tt>** node and rename it as `SessionLayerTargetSet`. Connect the **stage** and **sessionLayer** output plugs from the `SessionLayerGet` node to the **stage** and **layer** input plugs, respectively, of the new node. The board should look like the following:
```

Comment:
```suggestion
7. Setting the edit target layer of the stage is important so that any modification done to the stage is registered in that layer instead of the root layer. Add a new **<tt>UsdStageEditTargetSet</tt>** node and rename it as `SessionLayerTargetSet`. Connect the **stage** and **sessionLayer** output plugs from the `SessionLayerGet` node to the **stage** and **layer** input plugs, respectively, of the new node. The board should look like the following:
```